### ISSUE TITLE
RoteZora CNC - Enclosure

### ISSUE THREAD

**Issue by Blacksheep70** [comakingspace/do-something#158]
We need to build an enclosure for the "RoteZora" CNC Machine to protect the users and bystanders

Current status:

- The existing enclosure made from Polycarbonate used for the turning lathe does not fit (to small in dimensions)
- Actual plan is to build a closet from thick chip board (presspanplatte) with polycarbonate doors sitting on a bench. Later we might decide to enforce the strength with applying sheets of steelpanels (sandwich layer)
- Bench needs a compartment on the bottom housing Controller, VFD and Computer
- Bench needs a VESA mount for Monitor with a holder for Keyboard and Mouse
- Dimensions needed: W1400 x D1100 x H1100
- Microswitches for the doors in the emergency stop circuit (open doors doesn't allow start milling)
- New emergency stop switch on the outside of the enclosure
- Michael will buy some chip board (expense carried by CoMakingspace?) to create the housing on next Wednessday. 
- For the front doors we might use the polycarbonate from the other housing?

**Comment by LukasFreeze** [comment]
Please check with Patrick *before* buying materials, we have a lot of MDF
and plywood in the storage room/"hallway" that may be usable :-)

Micha B. made the existing polycarbonate housing for woodturning
demonstrations - I think it would be nice to keep that option, would the
pieces need to be cut to a different size to function as doors or could
they still be used in the current setup once or twice a year if you use
them? (Not sure how expensive they are anyway...)

**Comment by Blacksheep70** [comment]
A said I would use thick chip board (at least 25mm for the housing. I wonder if we have 

- 2x 1400x1100x25
- 2x 1100x1100x25
- 1x 1400x1100x25

available in the space. Would be great if someone could check that until Tuesday who is in the Space

I fear the polycarbonate housing is lost if we use it for the CNC housing doors. So probably we need to buy  1x1400x1100 polycarbonate too for the doors if you need to keep the housing.

The m² for the wood is at 10€. You know what? I buy the chip board on my own cost. I want to get this done asap.

**Comment by reogaro** [comment]
I left 2 big (not-so-micro)switches at the machine for securing the door.

**Comment by Blacksheep70** [comment]
Actually open questions where I need decisions:

- Overhauling
Johannes made the suggestion to overhaul the machine before bring it to "production" status
a) Replacing the cables
b) Adding protective "Schütz" switch
c) Adding a "energy chain" cable cover

Shall we do this before or should we bring the machiene productive anyway and do the overhaul later? I would vote for doing this later (I will take care of this). The machine is basically useable without it and this is just a "beautifying" task. If we come to the conclusion that we need to do this before setting the machine to production I would delay the work to the time after moving into the new lociation. 

- Housing 
I buyed the OSB boards for the housing and place them in the machining room. The housing will be quite larger than the machine to have enough room to handle the machine. The question here is where to place the machine in the old room.
If we decide to to the overhauling later I need a fix location where I can build up the machine.

**Comment by Pakue95** [comment]
Some of the „Overhaul“ items I’d say are safety features and should be done before clearing the machine for use. Also we don’t have a large amount of people who could realistically use the CNC anyways and those are fed up with the move. Therefore I’d say there is no harm in finishing the machine in the new location. If you come by the new space we can show you the spot we had in mind to put the CNC in.

**Comment by NitramLegov** [comment]
I would agree to Patrick. For me, this is a nice to have machine and not a top priority right now for anyone who can help with the move.

In addition, replacing cables, adding a schütz and a energy chain cable cover sound very much like safety features. If such a machine is supposed to be used by a broader audience (as in a makerspace), these sound like must-haves for me.

Therefore, I also vote for overhauling the machine first and make it available afterwards. If this means the machine will be available only after the move, that shouldn´t be a big deal.

In regards to the location: Please also have a look in the meeting minutes of the november Planum. Lukas already indicated the spot for the new cnc machine in the workshop area (lower left corner, right next to the office). This might drive your housing design.

**Comment by kenomaerz** [comment]
Falls es hier Bedarfe gibt, die wir sinnvoll als Spende formulieren können: Bitte hier die benötigten Bauteile und deren Preis listen und mich verlinken. Neue Werkzeuge können wir über betterplace.org als Spende durch Mitglieder finanzieren.

**Comment by Blacksheep70** [comment]
@NitramLegov NitramLegov 

Es gibt noch den Bedarf für das Acry-Schutzglas für die Fronttüren. Es gab zwar Vorschläge diese aus irgendwelche vorhandenen Reste zusammen zu stellen aber ich würde eine "ordentliche Lösung" bevorzugen.

Genaue Maße kann ich gerade nicht sagen, aber ich denke es sind 3-4 qm

Mögliche Quelle: https://acrylglasplattenshop.de/product/transparent-klar-4-mm-acrylglas-xt-platte/

Das wären aus meiner Sicht die einzigen Kosten die noch anfallen. Alles andere hab ich schon gekauft bzw. sind nur noch Kleinteile. Die Ölflex-Leitungen für die geplante Neuverkabelung kaufe ich noch sobald der Lockdown rum ist.

**Comment by kenomaerz** [comment]
Könnte das mal jemand durchrechnen, damit ich einen Bedarf anlegen kann?

**Comment by NitramLegov** [comment]
https://coms-hd.slack.com/archives/C013WF5EFCM/p1610800126002500?thread_ts=1610747512.001800&cid=C013WF5EFCM

**Comment by kenomaerz** [comment]
Merci!
